agent/pkg/chassis/loadbalancer/consistenthash/hashring/serviceinstance.go

```diff
@@ -41,7 +41,7 @@ type ServiceInstance struct {
 
 // String gets service instance key
 func (si ServiceInstance) String() string {
-	// key format: Namespace.Name.InstanceIP
+	// key format: Namespace#Name#InstanceIP
```

Comment:
what's the relation of this modified content between this PR?

Reply:
comment not match source code, codes use `#` to split service info.
So need to modify the comments.